runner.py

```diff
@@ -578,6 +584,9 @@ def seed(s):
 def main():
   args = arguments()
   seed(args.seed)
+  outdir = os.path.join(args.outdir, datetime.today().strftime('%Y-%m-%d-%H:%M:%S'))
```

Comment:
Can you add another flag for making a new directory with the time, which is `store_true`? I still want the default behavior to write to `outputs/`, and this will always generate a new folder